src/server/api.php

```diff
@@ -1028,25 +1028,31 @@ function get_covidcast_meta() {
     $geo_types = extract_values($_REQUEST['geo_types'], 'str');
 
     if ($time_types !== null || $signals !== null || $geo_types !== null) {
-      $epidata = array_filter($epidata, function($row) use(&$time_types, &$signals, &$geo_types) {
```

Comment:
This seems overly complicated -- why not use `$epidata = array_values($epidata)` instead?